src/main/kotlin/no/skatteetaten/aurora/gobo/GoboLiveness.kt

```diff
@@ -0,0 +1,35 @@
+package no.skatteetaten.aurora.gobo
+
+import io.micrometer.core.instrument.MeterRegistry
+import io.micrometer.core.instrument.Tag
+import mu.KotlinLogging
+import org.springframework.stereotype.Component
+
+private val logger = KotlinLogging.logger {}
+
+data class ConnectionPoolProblem(val totalConnections: Double, val pendingConnections: Double, val tags: List<Tag>)
+
+@Component
+class GoboLiveness(private val meterRegistry: MeterRegistry) {
+
+    val nettyTotalConnections = "reactor.netty.connection.provider.total.connections"
+    val nettyPendingConnections = "reactor.netty.connection.provider.pending.connections"
+
+    fun getConnectionPoolProblems() = meterRegistry
+        .find(nettyTotalConnections)
+        .gauges()
+        .mapNotNull { total ->
+            val pending = meterRegistry
+                .find(nettyPendingConnections)
+                .tags(total.id.tags)
+                .gauge()?.value() ?: 0.0
+
+            if (total.value() >= 15 && pending > 0) {
```

Comment:
Kanskje grenseverdi for pending skulle vært liiitt høyere enn 0? Vi kan jo finjustere dette etterhvert.

Reply:
gjort de verdiene konfigurerbare, har satt defaulten til 14 på total og 2 på pending.